cookbook/configuration/web_server_configuration.rst

```diff
@@ -310,10 +316,10 @@ The **minimum configuration** to get your application running under Nginx is:
 .. tip::
 
     This executes **only** ``app.php``, ``app_dev.php`` and ``config.php`` in
-    the web directory. All other files will be served as text. You **must**
-    also make sure that if you *do* deploy ``app_dev.php`` or ``config.php``
-    that these files are secured and not available to any outside user (the
-    IP address checking code at the top of each file does this by default).
+    the web directory. All other files will be denied. You **must** also make 
+    sure that if you *do* deploy ``app_dev.php`` or ``config.php`` that these 
+    files are secured and not available to any outside user (the IP address 
+    checking code at the top of each file does this by default).
```

Comment:
this last sentence no longer makes sense, if you do deploy app_dev.php or config.php, a 404 is returned so you don't have to secure it.